go/vt/wrangler/reparent.go

```diff
@@ -363,7 +363,11 @@ func (wr *Wrangler) plannedReparentShardLocked(ctx context.Context, ev *events.R
 	if topoproto.TabletAliasEqual(shardInfo.MasterAlias, masterElectTabletAlias) {
 		return fmt.Errorf("master-elect tablet %v is already the master", topoproto.TabletAliasString(masterElectTabletAlias))
 	}
-	oldMasterTabletInfo, ok := tabletMap[*shardInfo.MasterAlias]
+	var oldMasterTabletInfo *topo.TabletInfo
+	ok = false
```

Comment:
Ah makes sense, thanks for the reminder.

Then the error message below is gonna panic, so maybe special-case the nil case and return a specific error would be best.